scripts/zainscrape_classutil.py

```diff
@@ -0,0 +1,103 @@
+import sys
+import os
+import re
+import requests
+
+# Class to handle a timetable
+class timetable:
+    def __init__(self, courseCode, semester):
+        self.classes = extract(courseCode, semester)
+                self.selected = {}
+                for k in self.classes:
+                    self.selected[k] = None
+        def generateTimetable(self):
+            return None
+
+#removes tags from a line leaving just the information
+def clean(input):
+    result = re.sub(r'\<[^>]*\>','',input)
```

Comment:
Should also parse list tags since it is breaking descriptions?
E.g.
introduction to ethicsworkable ethical theoriesmorality of breaking the lawprivacypublic information and recordsand related topics Students carry out a significant software development project in teams.

Reply:
note the name of the file .. this is zain's classutil scraper. I haven't actually used it yet :/ maybe I should remove it from the repo for now...

Reply:
@PapaPwn but yes it is broken... i'll have to change the python and re run the scraper :'(

Reply:
Right, I thought you had been modifying and using it. All g for now.